src/ontology/mondo-ingest.Makefile

```diff
@@ -163,6 +163,23 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+# TODO: do
+# TODO: remove any unnecessary queries / lines here
+#  - fix_hgnc_mappings
+#  - maybe make a comment about other ones i thought to delete
+$(COMPONENTSDIR)/icd11.owl: $(TMPDIR)/icd11_relevant_signature.txt | component-download-icd11.owl
```

Comment:
## Analyze unmapped entities
- [ ] @joeflack4

I wonder if the easiest way to do this is by creating a SSSOM first when the ingest is working correctly.

_[Nico wrote](https://github.com/monarch-initiative/mondo-ingest/pull/412#discussion_r1485632153)_:
> It would be good to get a sense of which entities are not mapped after these 7 namespaces are applied.

<details><summary>Details</summary>
<p>

```
rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
rdfs: http://www.w3.org/2000/01/rdf-schema#
owl: http://www.w3.org/2002/07/owl#
skos: http://www.w3.org/2004/02/skos/core#
icd11.foundation: http://id.who.int/icd/entity/
icd11.schema: http://id.who.int/icd/schema/
icd11.z: http://who.int/icd#Z_
```

</p>
</details>

Reply:
@matentzn I'm just coming back to to this after a few weeks. Can you clarify again for me what you mean by the above quote? What is my action item?

Reply:
- [ ] Find: Are there any entities in the ontology that don't fall within these 7 namespaces. @joeflack4 

What is the quickest way to find such an entity?
One hacky way, 5 min without pain: Get all the entities in the file, OAK --terms, etc. Do a regex remove from each of these namespaces. or file sort it like as a TSV and look through.

Reply:
@matentzn LGTM:
I looked at this today. I ran `runoak -i INPUT terms > OUTPUT.csv`, then I just opened in Excel and sorted the column and looked at the unique namespaces that way.

- `icd11.foundation`: ~94k
- `icd11.schema`: 82
- `icd11.z:RetiredClasses`: 1
- `obo:mondo-ingest/components/icd11foundation.owl`: 1 _(seems weird but I'm not worried about it. I also don't see anything like this in `icd11foundation.owl`)_
- `skos`: 5 _(IDK why this appears here and e.g. `rdfs` doesn't, but I'm not worried about this either)_

Reply:
Trish and I looked today and verified that this has been addressed aadequately. Marking resolved.